servicetalk-grpc-gradle-plugin/src/main/groovy/io/servicetalk/grpc/gradle/plugin/ServiceTalkGrpcPlugin.groovy

```diff
@@ -80,26 +79,23 @@ class ServiceTalkGrpcPlugin implements Plugin<Project> {
           // uber jar which contains the protoc logic, as otherwise the grpc-gradle-plugin will only add a dependency
           // on the executable script
           File uberJarFile
-          String scriptNamePrefix
```

Comment:
the rational for putting the version suffix was that if folks update their version of ServiceTalk and the script generation changes, they won't have to clean their build in order to get the updates. Can you clarify the motivation for this change?

Reply:
This is not an issue if we generate a new file every time (see motivation described in the discussion below).
When you run project with `--include-build` the version is taken from `project.version` variable instead of `implementation-version` property. But `project.version` with `--include-build` is equal to the version of the project that runs with `--include-build`, it's not a ST version. For example, if I have a project with version `5.5` and run it with `--include-build=/path/to/servicetalk` I will see `5.5` in the script name.